crates/ruff_python_formatter/src/other/parameters.rs

```diff
@@ -63,8 +62,41 @@ impl FormatNodeRule<Parameters> for FormatParameters {
 
         let comments = f.context().comments().clone();
         let dangling = comments.dangling_comments(item);
+
         let (slash, star) = find_argument_separators(f.context().source(), item);
 
+        // First dangling comment: trailing the opening parenthesis, e.g.:
+        // ```python
+        // def f(  # comment
+        //     x,
+        //     y,
+        //     z,
+        // ): ...
+        // TODO(charlie): We already identified this comment as such in placement.rs. Is there any
+        // way to avoid this implicit coupling?
+        let parenthesis_comments_end = usize::from(dangling.first().is_some_and(|comment| {
+            if comment.line_position().is_end_of_line() {
+                // Ensure that there are no tokens between the open bracket and the comment.
+                let mut lexer = SimpleTokenizer::new(
+                    f.context().source(),
+                    TextRange::new(item.start(), comment.start()),
+                )
+                .skip_trivia()
+                .skip_while(|t| {
+                    matches!(
+                        t.kind(),
+                        SimpleTokenKind::LParen
+                            | SimpleTokenKind::LBrace
+                            | SimpleTokenKind::LBracket
+                    )
+                });
```

Comment:
Unrelated to this PR but just popped to my head (may be relevant for other places where we apply the same logic). What happens if we have: 

```Python
call(( a # test
)) 
```

I would expect this to be a comment of `a` and not a dangling arguments comment.